frontend/app/routes/protected/person-case/state-machine.server.ts

```diff
@@ -95,8 +96,17 @@ export const machine = setup({
       target: '.privacy-statement',
       actions: assign(initialContext),
     },
+    exit: {
+      target: '.exited',
+      actions: assign(initialContext),
+    },
   },
   states: {
+    'exited': {
+      meta: {
+        route: 'routes/protected/index.tsx',
+      },
+    },
```

Comment:
I'm not sure if this will affect the execution, but I think this state should have `type: 'final'` as covered [here](https://stately.ai/docs/final-states).

Reply:
By using the final state, it should clean up the context and delete any data.
In this case, your route wouldn't use the `meta` value, but instead just end the exit event and then redirect to the target page.

Reply:
basically:
- remove `exit.actions`
- remove `states.exited.meta`
- remove the `remove tid search param code`
- in the action do this:
  ``` ts
  case 'back': {
    machineActor.send({ type: 'exit' });
    throw i18nRedirect('routes/protected/index.tsx', request);
  }
  ```